index.html

```diff
@@ -116,9 +116,67 @@ <h1>Badge</h1>
         src="https://robohash.org/127539fe860f082d7fb9e34dbf58d4ed?set=set4&bgset=bg1&size=400x400"
         alt="avatar"
       />
-      <span class="badge has-bg-secondary is-bottom-right">10</span>
+      <span class="badge is-secondary is-bottom-right">10</span>
     </figure>
 
     <button>Badge in button <span class="badge">10</span></button>
+
+    <hr />
+    <h1>Alert</h1>
+    <hr />
+
+    <div class="alert is-primary" role="alert">
+      <strong>A simple primary alert—check it out!</strong>
+    </div>
+    <div class="alert is-secondary" role="alert">
+      <strong>A simple secondary alert—check it out!</strong>
+    </div>
```

Comment:
I think classes should be named like primary-alert or alert-primary instead of is-primary.
Nice that you added role attribute.